prowler/providers/aws/services/kms/kms_cmk_not_deleted_unintentionally/kms_cmk_not_deleted_unintentionally.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "kms_cmk_not_deleted_unintentionally",
+  "CheckTitle": "AWS KMS keys should not be deleted unintentionally",
+  "CheckType": [
+    "Data Deletion Protection"
+  ],
+  "ServiceName": "kms",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:kms:region:account-id:certificate/resource-id",
+  "Severity": "critical",
+  "ResourceType": "AwsKmsKey",
+  "Description": "Ensure there is no customer keys scheduled for deletion.",
+  "Risk": "KMS keys cannot be recovered once deleted, also, all the data under a KMS key is also permanently unrecoverable if the KMS key is deleted.",
+  "RelatedUrl": "https://docs.aws.amazon.com/securityhub/latest/userguide/kms-controls.html#kms-3",
```

Comment:
Could you put this link in Remediation.Other and here a link to KMS docs.

Reply:
Done here https://github.com/prowler-cloud/prowler/pull/4595/commits/de5b08518e601e92cda14ee3ff9fd932d502dbaf